forms/static/forms/admin/dependency_fields.js

```diff
@@ -0,0 +1,27 @@
+(function($){
+    $(document).ready(function() {
+        let newspaperperson = $('#id_newspaperperson_set-0-victim_or_survivor_1:checked').length
```

Comment:
Lets use same coding standard (naming, spacing, etc.) as the other js files... I think they're using snake_case, etc.